scripts/haas_network.service

```diff
@@ -0,0 +1,19 @@
+[Unit]
+Description=HaaS Network Server 
+After=network.target
+Requires=httpd.service
```

Comment:
Please remove the dependency on the httpd service, as there's no requirement for httpd to be running before the network action service runs.

Is it possible to conditionally add a dependency on the database service if it's there? The database could be running elsewhere as well or it could be using sqlite.